### PR TITLE
scripts: colorize parsed dictionary logging

### PR DESCRIPTION
**Pull request by elliot-wdtl**
Print parsed dictionary based logging with color

Signed-off-by: Elliot Revell-Nash <elliot.revellnash@wdtl.com>

### CODE REVIEW

**Comment by elliot-wdtl** [comment]
I wasn't able to run the log parser script without changing the module name from "parser" to something else.

File: scripts/logging/dictionary/parser/log_parser_v1.py [hunk 1]
```diff
@@ -333,14 +334,14 @@ def print_hexdump(hex_data, prefix_len):
                 chr_vals += " "
 
             elif chr_done == HEX_BYTES_IN_LINE:
-                print("%s%s|%s" % ((" " * prefix_len), hex_vals, chr_vals))
+                print(f"{color}%s%s|%s{Fore.RESET}" % ((" " * prefix_len), hex_vals, chr_vals))
```
[the remark this answers is not in the document]
Comment (dcpleung) [reply scripts/logging/dictionary/parser/log_parser_v1.py:337]: If pylint doesn't complain, it's fine. no need to change.

**Review by dcpleung — APPROVED** [review]

**Comment by cfriedt** [comment]
@elliot-wdtl - please rebase and resolve conflicts

**Review by cfriedt — APPROVED** [review]
LGTM

**Pull request merged**